Docs/reference/content/reference/driver/crud/compression.md

```diff
@@ -0,0 +1,108 @@
++++
+date = "2020-01-29T22:05:03-04:00"
+title = "Compression"
+[menu.main]
+  parent = "Reference Reading and Writing"
+  identifier = "Compression"
+  weight = 25
+  pre = "<i class='fa'></i>"
++++
+
+## Compression
+
+The C# driver supports compression of messages to and from MongoDB servers. The driver implements the three algorithms that are supported by MongoDB servers:
+
+* [Snappy](https://google.github.io/snappy/): Snappy compression can be used when connecting to MongoDB servers starting with the 3.4 release.
+* [Zlib](https://zlib.net/): Zlib compression can be used when connecting to MongoDB servers starting with the 3.6 release.
+* [Zstandard](https://facebook.github.io/zstd/): Zstandard compression can be used when connecting to MongoDB servers starting with the 4.2 release.
+
+The driver will negotiate which, if any, compression algorithm is used based on capabilities advertised by the server in the [ismaster]({{<docsref "reference/command/isMaster/">}}) command response. 
+
+### Specify compression via `URI`
```

Comment:
Replace `URI` with `Connection String` (unless that's inconsistent with other documentation)